src/TLuaInterpreter.cpp

```diff
@@ -3581,7 +3581,7 @@ int TLuaInterpreter::setTextFormat( lua_State * L )
 
 int TLuaInterpreter::raiseWindow( lua_State *L )
 {
-    string luaSendText="";
+    QString windowName = "";
```

Comment:
You don't have to initialise `QStrings` - they are automatically created containing a _null_ value which is indistinguishable from an _empty_ QString  except that `QString::isNull()` and `QString::isEmpty()` both return `true` for the former but an empty string gives `true` only for the second test.